drivers/console/ram_console.c

```diff
@@ -29,7 +29,10 @@
 #define RAM_CONSOLE_BUF_ATTR
 #endif
 
+#ifndef CONFIG_RAM_CONSOLE_BUFFER_SECTION
 char ram_console_buf[CONFIG_RAM_CONSOLE_BUFFER_SIZE] RAM_CONSOLE_BUF_ATTR;
+#endif
+
```

Comment:
revert this and directly use `extern char ram_console_buf[];` in lib/open-amp/resource_table.c.